### ISSUE TITLE
Use decimal instead of double

### ISSUE THREAD

**Issue by jangruenwaldt** [zuvola/flutter_simple_calculator#9]
Double is super inaccurate

**Comment by zuvola** [comment]
You're absolutely right.
However, I don't plan to implement it for a while as I don't have the time to do so.

**Comment by jangruenwaldt** [comment]
> You're absolutely right. However, I don't plan to implement it for a while as I don't have the time to do so.

Yeah, I totally get it. I dont really have time either. Still a great package. I'm fine with double for now as well.
